### PR TITLE
BETA: Register jagad .is-a.dev

### PR DESCRIPTION
**Pull request by jagadrenata**
Added `jagad .is-a.dev` using the [dashboard](https://manage.is-a.dev).

### CODE REVIEW

File: domains/jagad .json
Comment (Stef-00012) [inline domains/jagad .json:1]: Invalid file name, can not contain spaces

File: domains/jagad-r.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "Jagadoye",
+        "email": "ezzbro121@gmail.com"
+    },
+    "record": {
+        "A": ["147.56.097.097"]
```
Comment (Stef-00012) [inline domains/jagad-r.json:7]: Invalid IP address

File: domains/jgd.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "Jagadoye",
+        "email": "ezzbro121@gmail.com"
+    },
+    "record": {
+        "URL": "github.com/jagadoye/jagadoye.github.io"
```
Comment (Stef-00012) [inline domains/jgd.json:7]: Missing protocol (https://)

File: domains/my-domain.json [hunk 3]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "Jagadoye",
+        "email": "ezzbro121@gmail.com"
+    },
+    "record": {
+        "CNAME": "github.com/jagadoye/jagadoye.github.io"
```
Comment (Stef-00012) [inline domains/my-domain.json:7]: CNAME can not contain /

**Review by Stef-DP — CHANGES_REQUESTED** [review]

**Comment by orxngc** [comment]
Inactive

**Pull request closed without merging**